cyclonedx-lib/build.xml

```diff
@@ -107,12 +107,35 @@
         <target name="jar">
                 <mkdir dir="build/jar"/>
                 <jar destfile="build/jar/temurin-gen-sbom.jar" basedir="build/classes">
-                        <manifest>
-                                <attribute name="Main-Class" value="temurin.sbom.TemurinGenSBOM"/>
-                        </manifest>
+                <manifest>
+                <attribute name="Main-Class" value="temurin.sbom.TemurinGenSBOM"/>
+                </manifest>
                 </jar>
         </target>
 
+        <target name="buildSignSBOM">
+                <mkdir dir="build/classes"/>
+                <javac srcdir="src" destdir="build/classes" classpath="build/jar/cyclonedx-core-java.jar" includeantruntime="false"/>
+                <javac debug="true" debuglevel="lines,vars,source" srcdir="src" destdir="build/classes" classpath="build/jar/cyclonedx-core-java.jar" includeantruntime="false"/>
+                </target>
+
+        <target name="runSignSBOM">
+                <exec command="openssl req -x509 -newkey rsa:2048 -keyout RSAPrivate.pem -out cert.pem -days 365 -nodes"/>
```

Comment:
Don't think you need this line and cert.pem
You just need the the private key and a public key from it eg:
```
openssl genrsa -des3 -passout pass:test -out RSAPrivate.pem 2048
openssl rsa -in RSAPrivate.pem -passin pass:test -pubout -out RSAPublic.pub
```